tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -2091,4 +2091,63 @@ class CompletionSuite extends BaseCompletionSuite {
     }
   )
 
+  val BacktickCompletionsTag =
+    IgnoreScala3.and(IgnoreScalaVersion.forLessThan("2.13.17"))
```

Comment:
This is because the change needs https://github.com/scala/scala/pull/10986 to work. I wrote these tests by publishing locally patched compiler with those upstream changes.